projects/storefrontlib/shared/components/list-navigation/total/total.component.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * SPDX-FileCopyrightText: 2022 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { PaginationModel } from '@spartacus/core';
+
+@Component({
+  selector: 'cx-total',
+  templateUrl: './total.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TotalComponent {
+  private _pagination: PaginationModel;
+
+  get pagination(): PaginationModel {
+    return this._pagination;
+  }
+  @Input() set pagination(value: PaginationModel | undefined) {
```

Comment:
I would name the `value` parameter with a name that explains its role or content
You might also want to use the syntax `elvis` here? :)  Up to you - I simply like it :)